java.lang.NoClassDefFoundError: okhttp3/Interceptor

```java
java.lang.NoClassDefFoundError: okhttp3/Interceptor
    at com.google.maps.GeoApiContext$Builder.<init>(GeoApiContext.java:389)
Caused by: java.lang.ClassNotFoundException: okhttp3.Interceptor
    at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
    at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:522)
    ... 9 more
```

#### Environment details

1. Java 11
2. Mac OS X 12.3.1
3. `com.google.maps:google-maps-services:2.1.0`

#### Code example

```java
new GeoApiContext.Builder().apiKey("...")
```


#### Additional information
The issue does not exist on `com.google.maps:google-maps-services:2.0.0`

--- a/build.gradle
+++ b/build.gradle
@@ -52,12 +52,12 @@ artifacts {
 
 dependencies {
     compileOnly 'com.google.appengine:appengine-api-1.0-sdk:2.0.4'
-    api 'com.squareup.okhttp3:okhttp:5.0.0-alpha.7'
+    api 'com.squareup.okhttp3:okhttp:4.10.0'
     api 'com.google.code.gson:gson:2.9.0'
     api 'io.opencensus:opencensus-api:0.31.0'
     implementation 'org.slf4j:slf4j-api:1.7.36'
     testImplementation 'junit:junit:4.13.2'
-    testImplementation 'com.squareup.okhttp3:mockwebserver:5.0.0-alpha.7'
+    testImplementation 'com.squareup.okhttp3:mockwebserver:4.10.0'
     testImplementation 'org.apache.httpcomponents:httpclient:4.5.13'
     testImplementation 'org.slf4j:slf4j-simple:1.7.36'
     testImplementation 'org.apache.commons:commons-lang3:3.12.0'

